src/main/java/org/zalando/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -56,21 +63,51 @@ public Subscription createSubscription(final SubscriptionBase subscriptionBase)
             throw new InternalNakadiException("Serialization problem during persistence of event type", e);
         } catch (final DuplicateKeyException e) {
             throw new DuplicatedSubscriptionException("Subscription with the same key properties already exists", e);
+        } catch (final DataAccessException e) {
+            LOG.error("Database error when creating subscription", e);
+            throw new ServiceUnavailableException("Error occurred when running database request");
         }
     }
 
-    public Subscription getSubscription(final String id) throws NoSuchSubscriptionException {
+    public Subscription getSubscription(final String id) throws NoSuchSubscriptionException,
+            ServiceUnavailableException {
         final String sql = "SELECT s_subscription_object FROM zn_data.subscription WHERE s_id = ?";
         try {
             return jdbcTemplate.queryForObject(sql, new Object[]{id}, rowMapper);
         } catch (final EmptyResultDataAccessException e) {
             throw new NoSuchSubscriptionException("Subscription with id \"" + id + "\" does not exist", e);
+        } catch (final DataAccessException e) {
+            LOG.error("Database error when getting subscription", e);
+            throw new ServiceUnavailableException("Error occurred when running database request");
+        }
+    }
+
+    public List<Subscription> listSubscriptions() throws ServiceUnavailableException {
+        try {
+            return jdbcTemplate.query("SELECT s_subscription_object FROM zn_data.subscription ORDER BY " +
+                    "s_subscription_object->>'created_at' DESC", rowMapper);
+        } catch (final DataAccessException e) {
+            LOG.error("Database error when listing subscriptions", e);
+            throw new ServiceUnavailableException("Error occurred when running database request");
+        }
+    }
+
+    public List<Subscription> listSubscriptionsForOwningApplication(final String owningApplication)
+            throws ServiceUnavailableException {
+        final String query = "SELECT s_subscription_object FROM zn_data.subscription " +
```

Comment:
We have queries like:
final String sql = "SELECT s_subscription_object FROM zn_data.subscription WHERE s_id = ?";
and like:
final String query = "SELECT s_subscription_object FROM zn_data.subscription " +
 +                "WHERE s_subscription_object->>'owning_application' = ? " +
 +                "ORDER BY s_subscription_object->>'created_at' DESC";

I mean that in one query we use  s_subscription_object->>'owning_application' = ?  but in others we use just s_id = ? instead of s_subscription_object->>'s_id' = ?.
I do not know the difference in that.

Maybe it is just about the style?